pmd.properties

```diff
@@ -1,12 +1,20 @@
 category=External Analysers
 description=Provide PMD rules to analyse Java projects
 homepageUrl=https://github.com/jborgers/sonar-pmd
-archivedVersions=3.4.0,3.5.0
-publicVersions=3.5.1
+archivedVersions=3.5.0,3.5.1
+publicVersions=4.0.0
 
 defaults.mavenGroupId=org.sonarsource.pmd
 defaults.mavenArtifactId=sonar-pmd-plugin
 
+4.0.0.description=PMD 7.10.0, supports Java 24-preview, runs on Java 11+ analysis side
+4.0.0.sqs=[10.8,LATEST]
+4.0.0.sqcb=[24.12,LATEST]
+4.0.0.sqVersions=[9.9.4,LATEST]
+4.0.0.date=2025-02-24
+4.0.0.changelogUrl=https://github.com/jborgers/sonar-pmd/releases/tag/4.0.0
+4.0.0.downloadUrl=https://github.com/jborgers/sonar-pmd/releases/download/4.0.0/sonar-pmd-plugin-4.0.0.jar
+
 3.5.1.description=PMD 6.55, support Java 20-preview (Java 21), run on Java 11+ analysis side
 3.5.1.sqs=[10.8,LATEST]
```

Comment:
Only one SQS version can be compatible with `LATEST` at a time

Reply:
Thanks, I just found out, should be ok now